keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -189,138 +189,76 @@ Yes. It is tested by `TestUpdateServiceLoadBalancerStatus` in pkg/registry/core/
 
-<!--
-This is your opportunity to define what "normal" quality of service looks like
-for a feature.
-
-It's impossible to provide comprehensive guidance, but at the very
-high level (needs more precise definitions) those may be things like:
-  - per-day percentage of API calls finishing with 5XX errors <= 1%
-  - 99% percentile over day of absolute value from (job creation time minus expected
-    job creation time) for cron job <= 10%
-  - 99.9% of /health requests per day finish with 200 code
-
-These goals will help you determine what you need to measure (SLIs) in the next
-question.
--->
+No increase of error rate when a workload of a cluster is targeting a service of 
+type LoadBalancer and the feature enabled.
 
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
-<!--
-Pick one more of these and delete the rest.
--->
-
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+- [X] Other (treat as last resort)
+  - Details: Workload/Application instrumentation containing the error rate and 
+  latency of calls against other services on this cluster
 
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
-<!--
-Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
-implementation difficulties, etc.).
--->
+N/A
 
 ### Dependencies
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### Does this feature depend on any specific services running in the cluster?
 
-<!--
-Think about both cluster-level services (e.g. metrics-server) as well
-as node-level agents (e.g. specific version of CRI). Focus on external or
-optional services that are needed. For example, if this feature depends on
-a cloud provider API, or upon an external software-defined storage or network
-control plane.
-
-For each of these, fill in the following—thinking about running existing user workloads
-and creating new ones, as well as about cluster-level services (e.g. DNS):
-  - [Dependency name]
-    - Usage description:
-      - Impact of its outage on the feature:
-      - Impact of its degraded performance or high-error rates on the feature:
--->
+- cloud controller manager /  LoadBalancer controller
+  - LoadBalancer controller should set the right .status field for `ipMode`
+    - In case of this feature outage, the traffic may still be routed using the `VIP` mode
+- kube-proxy
+  - Network interface IP addressprogramming
+    - In case of this feature outage, network interfaces on the node may still keep 
+    adding the LoadBalancer IP, that may cause wrong traffic routing
+    - This dependency doesn't happen on clusters that uses CNI that replaces kube-proxy. 
+    The CNIs should implement this feature their own, in this case.
```

Comment:
Again, the new feature has no relevant effect on _outages_; if the service proxy crashes, then services stop getting updated, but that's independent of whether you are using the new feature or not.

However, there is a question here of what happens if you run a CCM that tries to use the new feature, with a Service Proxy that doesn't implement the feature. (Fortunately, this is an easy answer: you get the same results you would have gotten if the CCM hadn't set `ipMode`.)